### ISSUE TITLE
Bash file to set up dev environment

### ISSUE THREAD

**Issue by norrishd** [utmittal/pyramidal-action-party#9]
This could get much more complicated over time, but for now it would be great just to have a bash file that just `pip install`ed all required Python packages.

Note this might be difficult/annoying to maintain with people working across 3 OSs (Nikunj/Utkarsh Windows (?), Liza OSX, Noosh Ubuntu).

**Comment by utmittal** [comment]
We could create multiple bash files. It's like any installation, different installer for every OS.

That said, commands might be the same for OSX and Ubuntu?

**Comment by norrishd** [comment]
Yeah that's surely the solution. I think bash commands are mostly interchangeable between osx/Ubuntu (apparently some flags are different), but directory structure is different
